Odd indent error after updating to 1.3.1

I'm pretty sure this was working in ESLint v1.2, but I just tried updating a large project to ESLint v1.3.1 and am getting a few odd `indent` rule errors:

### INPUT:
```js
var P = require('bluebird');

var obj = {
  send: function () {
    return P.resolve({
      type: 'POST'
    })
    .then(function () {
      return true;
    }, function () {
      return false;
    });
  }
};

obj.send().then(console.log).catch(console.error);
```

### .eslintrc
```yaml
extend: eslint:recommended

envs:
  node: true

rules:
  indent: [2, 2]
```

### OUTPUT:
```bash
➜  eslint-bluebird  npm test

> eslint-bluebird@1.0.0 test /Users/pdehaan/dev/tmp/del/eslint-bluebird
> eslint --version; eslint .

v1.3.1

index.js
  11:7  error  Expected indentation of 8 space characters but found 6  indent
  12:5  error  Expected indentation of 6 space characters but found 4  indent

✖ 2 problems (2 errors, 0 warnings)

npm ERR! Test failed.  See above for more details.
```

### WORKAROUND:
It seems like ESLint wants me to use this indenting, which looks a bit funky near the end:

```js
var P = require('bluebird');

var obj = {
  send: function () {
    return P.resolve({
      type: 'POST'
    })
    .then(function () {
      return true;
    }, function () {
        return false;
      });
  }
};

obj.send().then(console.log).catch(console.error);
```

--- a/lib/rules/indent.js
+++ b/lib/rules/indent.js
@@ -238,7 +238,7 @@ module.exports = function(context) {
         //     });
         var calleeLoc = calleeNode.type === "CallExpression" && calleeNode.callee.loc;
         if (calleeLoc &&
-            calleeNode.loc.start.line < calleeLoc.end.line &&
+            calleeLoc.end.line - calleeLoc.start.line === 1 &&
             calleeLoc.end.line < node.loc.start.line
         ) {
             indent += indentSize;
@@ -246,15 +246,17 @@ module.exports = function(context) {
 
         indent += indentSize;
         // If function content is not empty
+        var bodyIndent;
         if (node.body.length > 0) {
+            bodyIndent = getNodeIndent(node.body[0]);
             // Calculate left shift position don't require strict indent
             // allow function body align to (indentSize * X)
-            while (getNodeIndent(node.body[0]) > indent) {
+            while (bodyIndent > indent) {
                 indent += indentSize;
             }
-        }
 
-        checkNodesIndent(node.body, indent);
+            checkNodesIndent(node.body, indent);
+        }
 
         checkLastNodeLineIndent(node, indent - indentSize);
     }

--- a/tests/lib/rules/indent.js
+++ b/tests/lib/rules/indent.js
@@ -783,9 +783,25 @@ ruleTester.run("indent", rule, {
                 "      \n" +
                 "  }\n" +
                 "\n" +
-                "function hello () { // <-- eslint complains about this line\n" +
+                "function hello () {\n" +
                 "    \n" +
                 "}\n",
+            options: [2]
+        },
+        {
+            code:
+                "var obj = {\n" +
+                "  send: function () {\n" +
+                "    return P.resolve({\n" +
+                "      type: 'POST'\n" +
+                "    })\n" +
+                "    .then(function () {\n" +
+                "      return true;\n" +
+                "    }, function () {\n" +
+                "      return false;\n" +
+                "    });\n" +
+                "  }\n" +
+                "};\n",
             options: [2]
         }
     ],

